app/scheduling/directives/termCalendar.js

```diff
@@ -253,11 +323,24 @@ schedulingApp.directive("termCalendar", this.termCalendar = function ($rootScope
 				return calendarActivities;
 			};
 
-			var styleCalendarEvents = function (calendarActivities, backgroundColor, borderColor, textColor) {
+			var styleCalendarEvents = function (calendarActivities, backgroundColor, borderColor, textColor, tagColor) {
 				calendarActivities.forEach(function (event) {
-					event.color = (scope.view.state.uiState.selectedActivityId === event.activityId) ? highlightedEventBackgroundColor : backgroundColor;
-					event.borderColor = (scope.view.state.uiState.selectedActivityId === event.activityId) ? highlightedEventBorderColor : borderColor;
-					event.textColor = (scope.view.state.uiState.selectedActivityId === event.activityId) ? highlightedEventTextColor : textColor;
+					console.log(scope.view.state.uiState.selectedActivityId);
+					if (scope.view.state.uiState.selectedActivityId === event.activityId) {
+						if (tagColor) {
+							event.color = lightenOrDarkenColor(tagColor, -80);
```

Comment:
Define a constant for "-80" which is named to explain what it is, e.g. SELECTED_ACTIVITY_COLOR_SHIFT or something better.